free-1.md

```diff
@@ -1,49 +1,99 @@
 # Free project 1
 
+
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+_What need is met by your idea?_
+
+Many people who love music and want to play the piano accompaniment for a song cannot read guitar chord tabs, but can read piano sheet music.
+
+_Who are you helping? What is that person's experience like now?_
+
+My project would meet the need of those who want chords to be presented in the form of notes on a musical staff, instead of in the form of a guitar tab sheet. 
+Right now, to play the accompaniment of a song, a user has to:
+Look up the chords on a site like https://tabs.ultimate-guitar.com
+Read the letters and numbers that make up the name of a chord
+Understand what notes are in a chord, given its name 
+FIgure out how those chords can be played on the piano
+If I could help them, they would be able to directly translate from the name of a chord to seeing a depiction of the chord, dictated on a musical staff. 
```

Comment:
So is your language going to just take in a string (i.e. a chord name) and then output the notes in the chord? That seems a little small. Alternatively, you could take in a whole tab or chord progression and output the equivalent in piano chords, which seems better (in terms of scope) but like it would take a lot of work to do extraneous things, i.e. serialize tabs for example.